src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -1,5 +1,6 @@
 package seedu.addressbook.commands;
 
+import java.lang.reflect.Array;
```

Comment:
I'm not sure if this import is ever used but in the event if it is not used, it is a good practice to remove unused imports.